templates/page.html

```diff
@@ -76,27 +76,63 @@ <h3>Free</h3>
                   <div class="line output">Hello Jane Doe!</div>
                   <div class="line output">Hello Joe Bloggs!</div>
                 </div>
-                <figcaption class="figure-caption text-center"><a href="https://github.com/reanahub/reana-demo-helloworld">"Hello world"</a></figcaption>
+                <figcaption class="figure-caption text-center">
+                  <i class="fab fa-github"></i>
```

Comment:
@VMois FYI I have pushed a commit on top of yours that will (i) make example images clickable and (ii) add some breathing space after GitHub link,  so that it'll be easier to go to the example to learn about it. WDYT?

Reply:
Looks nice. Only "helloworld" link is missing (cause it is not an image) but it is okay. I tried adding a link but it breaks the layout.